api/migrations/versions/4923563b3a63_add_reponseID_outcome.py

```diff
@@ -0,0 +1,42 @@
+""" Add foreign key response_id to outcome
+
+Revision ID: 4923563b3a63
+Revises: bf4cb562e36c
+Create Date: 2024-12-07 19:07:18.021334
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '4923563b3a63'
+down_revision = 'bf4cb562e36c'
+
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column(
```

Comment:
In order to make this non-nullable, you'll have to start with this nullable column as you have it here, then run some SQL to update (and therefore set) the response_id for all existing outcome records, and then update the column to be non-nullable.